drivers/volume/portworx/portworx.go

```diff
@@ -1055,16 +1053,6 @@ func (p *portworx) StartVolumeSnapshotRestore(snapRestore *stork_crd.VolumeSnaps
 		return err
 	}
 
-	if snapType == crdv1.PortworxSnapshotTypeCloud {
```

Comment:
We should add a version check here to only support this from PX 2.3.0 onwards

Reply:
this one was unnecessary , we have that on https://github.com/libopenstorage/stork/pull/505/files/bed5a31a466da0fde2427037658b028ea9c787d1#diff-0aa9aa57d00bdd00603aae645329fb04R1061

Reply:
Ok, that should be changed to 2.3.0

Reply:
ok